test/integration/flags.go

```diff
@@ -31,19 +32,55 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+var startTimeout = flag.Int("timeout", 25, "number of minutes to wait for minikube start")
 var binaryPath = flag.String("binary", "../../out/minikube", "path to minikube binary")
 var globalArgs = flag.String("minikube-args", "", "Arguments to pass to minikube")
 var startArgs = flag.String("minikube-start-args", "", "Arguments to pass to minikube start")
 var mountArgs = flag.String("minikube-mount-args", "", "Arguments to pass to minikube mount")
 var testdataDir = flag.String("testdata-dir", "testdata", "the directory relative to test/integration where the testdata lives")
+var disableParallel = flag.Bool("disable-parallel", false, "run the tests squentially and disable all parallel runs")
```

Comment:
Flags should not be named negative, as it can cause confusing combinations, for instance, if someone sets `--disable-parallel=false`. How about just `parallel`, but set it to true by default?